tests/EnvironmentTest.php

```
@@ -5,306 +5,432 @@
+      $mock_provider->expects($this->exactly($expected_provider_contextualize_calls))
+        ->method('contextualize');
+    }
+
+    Environment::init(
+      contextualize: $contextualize,
+      providers: [$mock_provider],
+      contexts: [$mock_context]
+    );
+
+    $this->assertEquals(Environment::STAGE, getenv('ENVIRONMENT_TYPE'));
+  }
```

PHPMD (2.15.0)

The parameter $has_active_context is not named in camelCase. (undefined)

(CamelCaseParameterName)